src/openms/include/OpenMS/CHEMISTRY/RibonucleotideDB.h

```diff
@@ -100,6 +101,14 @@ namespace OpenMS
     */
     ConstRibonucleotidePtr getRibonucleotidePrefix(const std::string& seq);
 
+    /**
+       @brief Get the alternatives for an ambiguous modification code
+
+       @throw Exception::ElementNotFound if nothing was found
+    */
+    std::pair<ConstRibonucleotidePtr, ConstRibonucleotidePtr> getRibonucleotideAlternatives(const std::string& code);
```

Comment:
can you add this to the pyopenMS .pxd file?

Reply:
I can take a shot at this tomorrow if @hendrikweisser doesn't have any objections.

Reply:
@poshul: Thanks, that would be helpful. Are you okay to update the bindings for TheoreticalSpectrumGenerator[NA] as well?

Reply:
Yes, I can do that, though it probably won't happen until tomorrow.

Reply:
Okay, dumb Cython question, ConstRibonucleotidePtr is defined: 
`using ConstRibonucleotidePtr = const Ribonucleotide *;`
How do I include that in a pxd? Attempting to use the underlying type ie
`   libcpp_pair[ const Ribonucleotide *, const Ribonucleotide * ] getRibonucleotideAlternatives(libcpp_string & code) nogil except +`
 throws 'arg code wrong type'.

How do I let Cython know the correct type?

Reply:
Apparently none of the other functions of `RibonucleotideDB` have been wrapped so far. Does `const Ribonucleotide *` work as a return type when you wrap e.g. `getRibonucleotide`? Based on how `ResidueDB` is wrapped, I would expect it does.